operations/kubernetes/Kyverno.md

```diff
@@ -402,47 +402,455 @@ Example output:
-and [match/exclude](https://release-1-10-0.kyverno.io/docs/writing-policies/match-exclude/).
+   For more information on policy exception and matchings, refer to the Kyverno documentation at [Policy Exceptions](https://release-1-10-0.kyverno.io/docs/writing-policies/exceptions/)
+   and [match/exclude](https://release-1-10-0.kyverno.io/docs/writing-policies/match-exclude/).
+
+## What is new in the HPE CSM 1.7 release and above
+
+1. Kyverno is upgraded from 1.10.7 version to 1.13.4 version and is now available for customers as part of the HPE CSM 1.7 release.
+
+   This is a major upgrade with many new features and bug fixes. For a complete list, refer to the external [CHANGELOG](https://github.com/kyverno/kyverno/blob/main/CHANGELOG.md).
+
+2. Container image signature verification is enforced using Kyverno policy.
+
+3. Pod Security Policies (PSP) are removed and Baseline Pod Security Standards (PSS) are enforced using Kyverno Policies.
+
+### Container image signature verification is enforced using Kyverno policy
+
+   Container image signing and runtime verification policy by name `check-image` is delivered (through Kyverno
+   policy engine) as part of CSM 1.7 release in `Enforce` mode. (`Audit` only mode will log the policy violation
+   warning messages into the cluster report and as events whereas `Enforce` mode will block the resources if they fail signature verification)
+
+   By default, the `check-image` policy is shipped as a `ClusterPolicy`. Policy can be customized based on the end users environment.
+   In CSM 1.7, a cluster-wide mutation policy named `prepend-registry` is provided to prepend `registry.local/` to all the container images that are being used in the Kubernetes resources.
+   Also, a policy exception named `check-image-exceptions` is provided that can be used to provide exceptions from both `prepend-registry` and `check-image` policy. End users can modify the `check-image-exceptions` to add any exceptions.
+
+   For more information on the above mutation policy, refer to the [Mutation Policy](https://kyverno.io/policies/other/prepend-image-registry/prepend-image-registry/).
+   In CSM 1.7, Container image signature verification policy `check-image` is separated from `kyverno-policy` helm chart and is delivered as a new helm chart named `image-verification-policy`.
+   This new chart is deployed after the nexus deployment. This change is introduced as Kyverno doesn't have the intelligence to look into the pit node(until nexus is up) for the image signatures.
+   Kyverno will look into nexus due to the prepended 'registry.local/'.
+
+#### How to identify container image signature verification failures
+
+##### Checking Existing Resources
+
+Use the following command:
+
+`kubectl get polr -A | awk '$6 > 0'`
+
+This command filters the `Policy Reports (polr)` to list resources with at least one verification failure. Sample output:
+
+```bash
```

Comment:
Switch this to `text` instead of `bash`
```suggestion
```text
```

Reply:
Changed as per review comment.